ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -114,6 +114,9 @@
 /* Accessibility value for when a UISwitch is off */
 "Accessibility.TableViewCell.Switch.Off" = "Off";
 
+/* Format string for tab bar item accessbility labels. Format: "<Title>, <BadgeCount> items". Example: "Home, 5 items" */
+"Accessibility.TabBarItemView.LabelFormat" = "%@, %ld items";
```

Comment:
tabbaritem itself doesn't need to know about the index. it could be change. If you look at the current tabbarview, the VO index string is done by the OS without us appending index accessibility label or hint. I think The TabBar trait is just recognized and handle by VO correctly.

Reply:
This is not for the index. This format string is to announce the badge number on the tab bar item.
This matches what Apple does with the build-in UITabBar.

Reply:
ah sorry for misreading this comment.